$NetBSD: patch-tools_parser_Makefile,v 1.2 2017/08/23 17:53:45 jlam Exp $

make a libparser.a file to link with, to avoid filename conflicts

--- tools/parser/Makefile.orig	2017-08-12 23:18:24.000000000 +0000
+++ tools/parser/Makefile
@@ -0,0 +1,18 @@
+#
+
+LIBISPRIVATE=	yes
+
+# For use with bootstrap-mk-files, which doesn't support LIBISPRIVATE.
+MKLINKLIB=	no
+MKPIC=		no
+MKPROFILE=	no
+
+LIB=	parser
+
+SRCS=	amp.c att.c avctp.c avdtp.c avrcp.c bnep.c bpa.c capi.c		\
+	cmtp.c csr.c ericsson.c hci.c hcrp.c hidp.c l2cap.c lmp.c	\
+	obex.c parser.c ppp.c rfcomm.c sap.c sdp.c smp.c tcpip.c
+
+CPPFLAGS+= -I../..
+
+.include <bsd.lib.mk>
